mongonaut/templatetags/mongonaut_tags.py

```diff
@@ -26,6 +25,6 @@ def get_document_value(document, key):
         app_label = value.__module__.replace(".models", "")
         document_name = value._class_name
         url = reverse("document_detail", kwargs={'app_label': app_label, 'document_name': document_name, 'id': value.id})
-        return mark_safe("""<a href="{0}">{1}</a>""".format(url, value))
+        return mark_safe("""<a href="{0}">{1}</a>""".format(url, value.__unicode__()))
```

Comment:
Why not call `str(value)`?  Not usually a great idea to call the dunder methods.  If it's needed, there should be a comment stating why.